BUILD.bazel

```diff
@@ -1,6 +1,25 @@
+load("@buildifier_prebuilt//:rules.bzl", "buildifier")
```

Comment:
You should move this so the load isn't in the root package.

Reply:
Done